src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java

```diff
@@ -393,68 +393,75 @@ private ArgumentsBuilder createBaselineArgumentsForFuncotator(final String varia
+                        "tmp2.vcf",
                         b37Reference,
                         FuncotatorTestConstants.REFERENCE_VERSION_HG19,
                         GERMLINE_DATASOURCES_FOLDER,
                         true
                 },
-                {
-                        "0816201804HC0_R01C01.vcf",
-                        b37Reference,
-                        FuncotatorTestConstants.REFERENCE_VERSION_HG19,
-                        FuncotatorTestConstants.FUNCOTATOR_DATA_SOURCES_LOCAL_CLOUD_FOLDER,
-                        false
-                },
-                {
-                        "0816201804HC0_R01C01.vcf",
-                        b37Reference,
-                        FuncotatorTestConstants.REFERENCE_VERSION_HG19,
-                        FuncotatorTestConstants.FUNCOTATOR_DATA_SOURCES_REMOTE_CLOUD_FOLDER,
-                        false
-                },
-                {
-                        "hg38_test_variants.vcf",
-                        hg38Reference,
-                        FuncotatorTestConstants.REFERENCE_VERSION_HG38,
-                        LARGE_DATASOURCES_FOLDER,
-                        true
-                },
-                {
-                        "hg38_trio.vcf",
-                        hg38Reference,
-                        FuncotatorTestConstants.REFERENCE_VERSION_HG38,
-                        LARGE_DATASOURCES_FOLDER,
-                        true
-                },
-                {
-                        FuncotatorTestConstants.NON_TRIVIAL_DATA_VALIDATION_TEST_HG19_DATA_SET_1,
-                        b37Reference,
-                        FuncotatorTestConstants.REFERENCE_VERSION_HG19,
-                        FuncotatorTestConstants.FUNCOTATOR_DATA_SOURCES_MAIN_FOLDER,
-                        false
-                },
-                {
-                        FuncotatorTestConstants.NON_TRIVIAL_DATA_VALIDATION_TEST_HG19_DATA_SET_2,
-                        b37Reference,
-                        FuncotatorTestConstants.REFERENCE_VERSION_HG19,
-                        FuncotatorTestConstants.FUNCOTATOR_DATA_SOURCES_MAIN_FOLDER,
-                        false
-                },
-                {
-                        FuncotatorTestConstants.NON_TRIVIAL_DATA_VALIDATION_TEST_HG38,
-                        hg38Reference,
-                        FuncotatorTestConstants.REFERENCE_VERSION_HG38,
-                        FuncotatorTestConstants.FUNCOTATOR_DATA_SOURCES_MAIN_FOLDER,
-                        false
-                },
-                {
-                        FuncotatorTestConstants.NON_TRIVIAL_DATA_VALIDATION_TEST_HG19_LARGE_DATA_SET,
-                        b37Reference,
-                        FuncotatorTestConstants.REFERENCE_VERSION_HG19,
-                        FuncotatorTestConstants.FUNCOTATOR_DATA_SOURCES_MAIN_FOLDER,
-                        false
-                },
+//                {
```

Comment:
Why are all of these tests being disabled?  If not being used, can you remove the code?

Reply:
Comments were for debugging and left in by mistake.